apicast/src/get_token.lua

```diff
@@ -10,7 +12,7 @@ local function extract_params()
   params.authorization = {}
 
   if header_params['Authorization'] then
-    params.authorization = ngx.decode_base64(header_params['Authorization']:split(" ")[2]):split(":")
+    params.authorization = split(ngx.decode_base64(split(header_params['Authorization']," ")[2]),":")
```

Comment:
This is a functional change which fixes the bug.
There used to be a global function `function string:split`, which didn't make it to APIcast, so `strvariable:split` is not a valid syntax.